examples/tensorflow/classification/configs/pruning_quantization/mobilenet_v3_imagenet_pruning_geometric_median.json

```diff
@@ -0,0 +1,48 @@
+{
+    "model": "MobileNetV3Small",
+    "pretrained": true,
+    "input_info": {
+        "sample_size": [1, 224, 224, 3]
+    },
+    "batch_size": 320,
+    "epochs": 65,
+    "optimizer": {
+        "type": "sgd",
+        "schedule_type": "multistep",
+        "schedule_params": {
+            "base_lr": 0.1,
+            "steps": [20, 40, 60],
+            "gamma": 0.1
+        },
+        "optimizer_params": {
+            "momentum": 0.9,
+            "nesterov": true
+        }
+    },
+    "dataset": "imagenet2012",
+    "dataset_type": "tfds",
+
+    "compression": [
+        {
+            "algorithm": "filter_pruning",
+            "pruning_init": 0.9,
```

Comment:
Is it just test changes for pruning rate?

Reply:
Yes, it's only for testing

Reply:
OK, I found here a lot of temporary changes, please, be sure you reverted them before merge PR.